content/blog/2025-06-02-invited-speakers/index.en.md

```diff
@@ -0,0 +1,124 @@
+Even better, have public information about how these things are handled at your event, as non guests will have similar problems!
+
+Ask the speaker for a title, abstract, bio and picture.
+Do not create or edit those without their explicit approval.
+
+If you want to record, ask speakers ahead of time and tell them what you want to do with the recording. 
+
+Clarify the speaking logistics with the speaker: in what kind of room will they speak? will they have access to a clicker? what are the power characteristics? what will be the slide aspect ratio?
+Do let them use their own laptop. 
+Do not expect them to have slides in a certain format for your AV team without telling them in advance. Better yet, just let them use their own laptop and tell them about available connections.
+
+## Be mindful of travel logistics!
+
+If you want a speaker to feature on your schedule, covering their travel and accommodation (T&A) is the number one way to help. 
+If you can't cover T&A, then… you can't afford having the speaker (unless you reach a special agreement with the speaker, whose employer might cover T&A!).
+
+Travel planning is a key piece of an in-person event. 
+Be mindful of your guest’s preferences and constraints regarding the booking process: do they prefer to book themselves and be reimbursed? 
+Does advancing money actually mean they pay taxes on the reimbursement? 
+Will they fly or travel on trains or use their car? 
+Your speaker’s job is probably to spend more time preparing their content than figuring out how not to lose money.
+
+Get to know your guest and trust that he/she knows the ins and outs of travel logistics better than you do. 
+Ask if they need to arrive a day earlier or leave a day later to take it into account at the T&A. 
+Trips of many hours plus time differences are tiring for many people. 
+Your speaker will be better if they sleep in a bed and not on a plane and will surely give a better talk if they are rested. 
+Take into account that some people live far away from major airports or train stations and probably need transportation between airports or airports/train-bus stations to get to your event. 
+Take those cost into account.
+
+If you pay for their hotel but they need to put a credit card on file for incidentials, do let them know ahead of time - and also how much. If the hotel puts a hold on that amount, ask if they can advance that and don't assume.
+
+## Be nice on site
+
+Offer support but don’t overdo it either. 
+Do they need tourism tips? Do they know how to get around? 
+Probably also something useful for any event participant.
+
+Don’t assume alcohol is equal to entertainment for everybody,
+or that everybody drinks alcohol.
+
+Thank the speaker for coming/participating.
+
+## Be a good session chair
+
+Don’t mess up their bio. 
```

Comment:
```suggestion
Confirm bios prior to announcing the speaker.
```

Reply:
Confirm bios prior to announcing the speaker and share about the speaker on social media. 

(We also mentioned about website, perhaps this is something to add above)